schema/src/project.yaml

```diff
@@ -70,11 +139,143 @@ slots:
   project_developer:
     range: ProjectRole
     inlined: false
+  id:
+    range: uriorcurie
+    identifier: true
+  image:
+    slot_uri: schema:image
+    description: an image.
+  address:
+    slot_uri: schema:address
+    description: an address.
+  showOnProjectPage:
+    range: boolean
+    description: Whether to show this organization on the project page.
+  marketType:
+    slot_uri: rfs:marketType
+    range: MarketTypeTypes
+    description: The type of market for the associated credits.
+  bioregion:
+    range: string
+    slot_uri: rfs:bioregion
+    description: The bioregion associated with the project.
+  biomeType:
+    range: string
+    slot_uri: rfs:biomeType
+    description: The type of biome associated with the project.
+  watershed:
+    range: string
+    slot_uri: rfs:watershed
+    description: The watershed associated with the project.
+  subWatershed:
+    range: string
+    slot_uri: rfs:subWatershed
+    description: The sub-watershed associated with the project.
+  ecosystemType:
+    slot_uri: rfs:ecosystemType
+    range: EnvironmentTypeTypes
+    description: The type of ecosystem associated with the project.
+    multivalued: true
+  environmentalAuthority:
+    slot_uri: rfs:environmentalAuthority
+    range: Organization
+    description: The environmental authority associated with the project.
+  administrativeArea:
+    range: AdministrativeArea
+    description: The administrative area associated with the project.
+  offchainCreditsInfo:
+    slot_uri: rfs:offchainCreditsInfo
+    range: OffchainCreditsInfo
+    description: Information about offchain credits associated with the project.
+  projectDuration:
+    slot_uri: rfs:projectDuration
+    range: string
+    description: The duration of the project.
+  ecologicalConnectivityIndex:
+    slot_uri: rfs:ecologicalConnectivityIndex
+    range: float
+    description: The ecological connectivity index of the project.
+  socialCulturalIndex:
+    slot_uri: rfs:socialCulturalIndex
+    range: float
+    description: The social cultural index of the project.
+  managementAreas:
+    slot_uri: rfs:managementAreas
+    range: ManagementArea
+    description: The management areas associated with the project.
+    multivalued: true
+    inlined: true
+  activity:
+    # TODO: associate with regen data standards activities
+    range: ActivityType 
+    description: the activity.
+    multivalued: true
```

Comment:
What `slot_uri` do we want for this?

Reply:
just added it as `rfs:activity` explicityly, I think by default it'll take your key and prefix it with your `default_prefix`